flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -357,6 +358,213 @@ public void testAlterTableWithPrimaryKey() throws TableNotExistException {
     Assert.assertEquals(properties, table("tl").properties());
   }
 
+  @Test
+  public void testAlterTableProperties() throws TableNotExistException {
+    sql("CREATE TABLE tl(id BIGINT) WITH ('oldK'='oldV')");
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("oldK", "oldV");
+
+    // new
+    sql("ALTER TABLE tl SET('newK'='newV')");
+    properties.put("newK", "newV");
+    Assert.assertEquals(properties, table("tl").properties());
+
+    // update old
+    sql("ALTER TABLE tl SET('oldK'='oldV2')");
+    properties.put("oldK", "oldV2");
+    Assert.assertEquals(properties, table("tl").properties());
+
+    // remove property
+    sql("ALTER TABLE tl RESET('oldK')");
+    properties.remove("oldK");
+    Assert.assertEquals(properties, table("tl").properties());
+  }
+
+  @Test
+  public void testAlterTableAddColumn() {
```

Comment:
Do we need to add a case to test when the name of the field to be added already exists?